recipes/evalml/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: evalml-core
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/evalml/evalml-{{ version }}.tar.gz
+  sha256: 1c4c440e9370f49ad21100908285a23e7c38fbc872feba34c0e6c47c39db8f76
+
+build:
+  number: 0
+
+outputs:
+  - name: evalml-core
+    build:
+      entry_points:
+          - evalml = evalml.__main__:cli
+      noarch: python
+    script: build_script.sh
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - numpy >=1.16.4
+        - pandas >=0.25.0
+        - scipy >=1.2.1,<1.5.0
+        - scikit-learn >=0.21.3,!=0.22,<0.23.0
+        - scikit-optimize
+        - tqdm >=4.33.0
+        - colorama
+        - cloudpickle >=0.2.2
+        - click >=7.0.0
+        - psutil >=5.6.3
+        - requirements-parser >=0.2.0
+        - shap >=0.35.0
+        - texttable >=1.6.2
+        - python >=3.6.*
+    test:
+      imports:
+        - evalml
+      requires:
+        - pytest >= 4.4.*
+        - nbval ==0.9.3
+        - python-graphviz >=0.8.4
+        - category_encoders >=2.0.0
+      source_files:
+        - evalml/*
+        - requirements.txt
+        - core-requirements.txt
+      commands:
+        - pytest evalml/tests --has-minimal-dependencies --deselect evalml/tests/component_tests/test_utils.py::test_all_components
+
+  - name: evalml
```

Comment:
Can you add `noarch: python` for this output too?

Reply:
@isuruf Like this?

```
  - name: evalml
    requirements:
      build:
        noarch: python
```